src/tui/table.rs

```diff
@@ -0,0 +1,53 @@
+use std::fmt::Display;
+
+use comfy_table::{presets::ASCII_MARKDOWN, Cell, ContentArrangement, Table};
+
+pub struct RusticTable {
+    table: Table,
```

Comment:
Do you know about the newtype pattern? i.e. you could easily use 
`pub struct RusticTable(Table)` and access table with `self.0`.

Or, maybe even better to just use an `type` definition and use the existing methods from `Table`?

Reply:
I removed abstraction table